src/styles/main.scss

```
@@ -0,0 +1,99 @@
+$main-font-size: 30px;
+$bg-color-day: #eee;
+$bg-color-day--hover: #ffbfcb;
+$gap-calendar: 1px;
+$height-day: 100px;
+$width-day: 100px;
+$min-days: 28;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+.page {
+  box-sizing: border-box;
```

If u want to set some parameters for the entire project - use `*` for this.
now It will not always work in your class.
use it in `reset.scss` file
```suggestion
  * {
    box-sizing: border-box;
  }  
  
  html {
    font-family: Arial, Helvetica, sans-serif;
  }
```